threedigrid_builder/grid/cross_section_definitions.py

```diff
@@ -151,17 +171,17 @@ def tabulate_egg(shape, width, height):
     widths = np.sqrt(p / q) * 2
 
     table = np.array([heights, widths]).T
-    return CrossSectionShape.TABULATED_TRAPEZIUM, width, height, table
+    return CrossSectionShape.TABULATED_TRAPEZIUM, width, height, table, None
 
 
 def tabulate_inverted_egg(shape, width, height):
     """Tabulate the egg shape, upside down.
 
     See tabulate_egg.
     """
-    type_, width, height, table = tabulate_egg(shape, width, height)
+    type_, width, height, table, yz = tabulate_egg(shape, width, height)
```

Comment:
Unused variable `yz`. It is customary to solve it like this (the variable `_` has no special meaning, it is just a convention in case you are not going to use the variable)

```suggestion
    type_, width, height, table, _ = tabulate_egg(shape, width, height)
```